### PR TITLE
ENG-15503: fix grammar-gen failures due to sqlcmd hanging

### PR DESCRIPTION
**Pull request by johnTcrawford**
... by adding 3 missing error messages that currently go unrecognized,
and modifying a fourth, plus some special code for CREATE VIEW
statements that just return 'null' (see ENG-15587); also, include the
current time in the final SUMMARY message.

### CODE REVIEW

File: tests/sqlgrammar/sql_grammar_generator.py [hunk 1]
```diff
@@ -1344,8 +1354,11 @@ def generate_sql_statements(sql_statement_type, num_sql_statements=0, max_save_s
                             ['Materialized view only supports INNER JOIN'],
                             ['windowed function call and GROUP BY in a single query is not supported'],
                             ['materialized view does not support self-join'],
-                            ['Windowed RANK function call expressions require an ORDER BY specification'],
```
Comment (kaikaizi) [inline tests/sqlgrammar/sql_grammar_generator.py]: Is this removal intentional?

Comment (johnTcrawford) [reply tests/sqlgrammar/sql_grammar_generator.py]: Yes, it's replaced with:
'Windowed', 'function call expressions require an ORDER BY specification'
which catches both:
'Windowed RANK function call expressions require an ORDER BY specification'
as before, and:
'Windowed DENSE_RANK function call expressions require an ORDER BY specification'
which I saw recently, as well as any other function name that might appear in that message.
